15-transformer.ipynb

```diff
@@ -448,13 +449,10 @@
    ],
    "source": [
     "max_len = 200\n",
-    "text = data.Field(sequential=True, fix_length=max_len, batch_first=True, lower=True, dtype=torch.long)\n",
-    "label = data.LabelField(sequential=False, dtype=torch.long)\n",
-    "datasets.IMDB.download('./')\n",
-    "ds_train, ds_test = datasets.IMDB.splits(text, label, path='./imdb/aclImdb/')\n",
-    "print('train : ', len(ds_train))\n",
-    "print('test : ', len(ds_test))\n",
-    "print('train.fields :', ds_train.fields)"
```

Comment:
Why did you remove these?

Reply:
So this way of importing data no long works in newest pytorch. If you try to do it in this way, it would give this error.
![image](https://user-images.githubusercontent.com/88493484/144694892-4c66ea20-8c19-4ebf-a852-b1f1c6c15ea7.png)
Instead, torchtext.legacy should be used if you want to achieve the same functionality.

Reply:
Oh, I just moved it to next block because of my personal convention of writing in jupyter. I will move it back.

Reply:
I will add these back. I just forgot to add them back when changing this block..